server/server.lua

```diff
@@ -37,19 +37,28 @@ local function tablelength(table)
     end
     return count
 end
-
+local function isAllowed(src)
+if not IsPlayerAceAllowed(src, 'command') then
```

Comment:
Fix the indentation to match the tabs of 4 spaces and add an argument to this function to check `command.argument` because not all commands require you to gave god permission